app/models/info_request.rb

```diff
@@ -53,7 +53,8 @@ class InfoRequest < ActiveRecord::Base
   @non_admin_columns = %w(title url_title)
 
   strip_attributes :allow_empty => true
-  strip_attributes :only => [:title], :replace_newlines => true
+  strip_attributes :only => [:title],
+                   :replace_newlines => true, :collapse_spaces => true
```

Comment:
I prefer this version as it gets me closer to what I would have intended had I copy-pasted

```
Information on the treatment
 of llamas at the Ministry of Silly Walks
```

(I would expect/like a single instead of a double space) but this might result in accidentally removing intentional extra whitespace so I've left this as a fixup so it's easy to get rid of it again. Thoughts welcome!